GitUI/CommandsDialogs/RevisionFileTree.cs

```diff
@@ -152,7 +152,7 @@ public void LoadRevision(GitRevision revision)
                 //restore selected file and scroll position when new selection is done
                 if (_revision != null)
                 {
-                    _revisionFileTreeController.LoadChildren(_revision, tvGitTree.Nodes, tvGitTree.ImageList.Images);
+                    await _revisionFileTreeController.LoadChildren(_revision, tvGitTree.Nodes, tvGitTree.ImageList.Images);
```

Comment:
❗️ Missing `ConfigureAwait` (I believe the argument in this case is `true`, assuming `TreeNode` is a UI control)